libbeat/docs/outputconfig.asciidoc

```diff
@@ -1186,3 +1211,30 @@ The following elliptic curve types are available:
 * P-256
 * P-384
 * P-521
+
+[[configuration-output-codec]]
+=== Output Codec Configuration
+
+Outputs that do not require a specific output encoding do support to change the encoding using the `codec` configuration. The codec configured must be one of `json` or `format`. By default the `json` codec is used.
+
+*`json.pretty`*: If `pretty` is set to true, events will be nicely formatted. The default is false.
```

Comment:
The formatting here isn't ideal. It would be better to have a definition list of options with the example indented after the option. I can fix this later if you want because it's hard to explain how to do this in review comments. It's Ok to publish as it is.